Description: 0001-use-external-libglee.patch
 fs-uae ships and links against its own version of libglee
 by default. We want to use the libglee available in Debian
 however and are therefore modifying the Makefile to make
 use of that.

--- fs-uae-2.2.3.orig/Makefile
+++ fs-uae-2.2.3/Makefile
@@ -62,7 +62,7 @@ cflags = $(common_flags) -std=c99 $(CFLA
 cxxflags = $(common_flags) $(CXXFLAGS)
 ldflags = $(LDFLAGS)
 libs = -L$(libfsemu_dir)/out -lfsemu `sdl-config --libs` \
-		`pkg-config --libs libpng` -lz
+		`pkg-config --libs libpng` -lz -lglee
 
 ifeq ($(devel), 1)
 	warnings = -Wno-unused-variable -Wno-unused-function -Wno-write-strings \
--- fs-uae-2.2.3.orig/libfsemu/Makefile
+++ fs-uae-2.2.3/libfsemu/Makefile
@@ -55,7 +55,7 @@ objects = obj/emu_emu.o obj/emu_video.o
 		obj/ml_sdl.o obj/ml_rawinput.o obj/ml_macosx.o obj/ml_x11.o \
 		\
 		obj/base.o obj/log.o obj/config.o obj/image.o obj/ref.o \
-		obj/glee.o obj/opengl.o obj/thread.o obj/string.o obj/filesys.o \
+		obj/opengl.o obj/thread.o obj/string.o obj/filesys.o \
 		obj/unicode.o obj/init.o obj/random.o obj/list.o obj/eventlog.o \
 		obj/queue.o obj/hashtable.o obj/inifile.o obj/time.o \
 		obj/lua_lapi.o \
